src/app/admin/food-event/createFoodEventForm.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+
+import { DateTime } from "luxon";
+import { SubmitHandler, useForm } from "react-hook-form";
+
+import { createFoodEvent } from "./userFoodEventActions";
+
+const HEADER_STYLES = "text-2xl my-8";
+const INPUT_STYLES =
+  "rounded-lg border w-full p-3 my-2 border-2 border-[#A689FF] focus:bg-[#ede9fe] ";
+const SUBMIT_STYLES =
+  "bg-awesomer-purple px-5 py-3 text-white rounded-md hover:bg-[#A689FF]";
+const CLEAR_STYLES =
+  "bg-white px-5 py-3 text-black rounded-md border border-[#94a3b8] hover:bg-[#eae5fa] hover:text-[#7055fd]";
+
+export declare type FormFields = {
+  mealType: string;
```

Comment:
I like the use of types here, however remember the meeting where we discussed using the generated Amplify types instead of creating our own copies? These will go out of date with the table if it changes. You can access the Amplify types by going `Schema["FoodEvent"]`. There are examples of this around